requirements.txt

```diff
@@ -1,5 +1,5 @@
 boto3>=1.9,<2.0
-liac-arff==2.3
+liac-arff==2.4
```

Comment:
thx for that, if openml requires it as well, maybe I should think about relaxing the version constraint.